packages/manager/src/features/LoadBalancers/LoadBalancerCreate/ServiceTargetDrawer.tsx

```diff
@@ -0,0 +1,353 @@
+      open={open}
+      title={isEditMode ? 'Edit Service Target' : 'Add a Service Target'}
+    >
+      <Typography>{SERVICE_TARGET_COPY.Description}</Typography>
+      <form onSubmit={formik.handleSubmit}>
+        <TextField
+          errorText={formik.errors.label}
+          label="Service Target Label"
+          name="label"
+          onChange={formik.handleChange}
+          value={formik.values.label}
+        />
+        <Autocomplete
+          textFieldProps={{
+            labelTooltipText: SERVICE_TARGET_COPY.Tooltips.Protocol,
+          }}
+          value={protocolOptions.find(
+            (option) => option.value === formik.values.protocol
+          )}
+          disableClearable
+          errorText={formik.errors.protocol}
+          label="Service Target Protocol"
+          onChange={(_, { value }) => formik.setFieldValue('protocol', value)}
+          options={protocolOptions}
+        />
+        <Autocomplete
+          onChange={(e, selected) =>
+            formik.setFieldValue('load_balancing_policy', selected.value)
+          }
+          renderOption={(props, option, state) => {
+            return (
+              <li {...props}>
+                <Stack flexGrow={1}>
+                  <Typography color="inherit">
+                    <b>{option.label}</b>
+                  </Typography>
+                  <Typography color="inherit">{option.description}</Typography>
+                </Stack>
+                <SelectedIcon visible={state.selected} />
+              </li>
+            );
+          }}
+          textFieldProps={{
+            labelTooltipText: SERVICE_TARGET_COPY.Tooltips.Algorithm,
+          }}
+          value={algorithmOptions.find(
+            (option) => option.value === formik.values.load_balancing_policy
+          )}
+          disableClearable
+          errorText={formik.errors.load_balancing_policy}
+          label="Algorithm"
+          options={algorithmOptions}
+        />
+        <Divider spacingBottom={24} spacingTop={24} />
+        <Typography sx={{ marginBottom: 2 }} variant="h3">
+          Endpoints
+        </Typography>
+        <EndpointTable
+          endpoints={formik.values.endpoints}
+          onRemove={onRemoveEndpoint}
+        />
+        <AddEndpointForm onAdd={onAddEndpoint} />
+        <Divider spacingBottom={12} spacingTop={24} />
+        <Stack alignItems="center" direction="row">
+          <Typography variant="h3">Service Target CA Certificate</Typography>
+          <TooltipIcon
+            status="help"
+            text={SERVICE_TARGET_COPY.Tooltips.Certificate}
+          />
+        </Stack>
+        <Typography>
+          <BetaChip
+            component="span"
+            sx={{ marginLeft: '0 !important', marginRight: '4px' }}
+          />
+          Upload service target endpoint CA certificates after the load balancer
+          is created and the protocol is HTTPS.{' '}
+          <Link to={AGLB_DOCS.Certificates}>Learn More.</Link>
```

Comment:
```suggestion
          <Link to={AGLB_DOCS.Certificates}>Learn more.</Link>
```
Stray capital letter
![Screenshot 2023-12-08 at 12 04 43 PM](https://github.com/linode/manager/assets/114685994/bb395f27-b37e-46c8-b003-672d5e0391b8)